doc/user/content/data-input-formats.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Data Input Formats"
```

Comment:
Is this section needed? This information will live in the refurbished landing page for sources that I'm pushing out this week(-ish)!